src/commands/positive.ts

```diff
@@ -106,6 +126,47 @@ export default class Positive extends ApiKeyCommand {
     }
   };
 
+  getSecurities = async (): Promise<string[]> => {
+    const operations = await this.schema.getOperations();
+
+    return uniq(
+      operations.flatMap((operation) => {
+        return operation.security.map((security) => security.key);
+      }),
+    );
+  };
+
+  promptForSecurityValues = async (flags: {
+    apiKey: string | undefined;
+  }): Promise<void> => {
+    const securitySchemes = await this.schema.getSecuritySchemes();
+    if (securitySchemes.length <= 0) {
```

Comment:
Minor:
Array length is defined as a non-negative integer, so there is no case in which it would be less than 0
https://stackoverflow.com/a/8190767